src/metabase/models/card.clj

```diff
@@ -325,3 +326,57 @@
 
   serdes.hash/IdentityHashable
   {:identity-hash-fields (constantly [:name (serdes.hash/hydrated-hash :collection)])})
+
+;;; ------------------------------------------------- Serialization --------------------------------------------------
+(defmethod serdes.base/extract-query "Card" [_ {:keys [user]}]
+  (serdes.base/raw-reducible-query
+    "Card"
+    {:select     [:card.*]
+     :from       [[:report_card :card]]
+     :left-join  [[:collection :coll] [:= :coll.id :card.collection_id]]
+     :where      (if user
```

Comment:
How exactly the `options` map for this extract-query are going to be created.
Is this going to be passed down from the serialization command?

Because if the options come from the command line, do we need to support an option where export _all_ cards?

Reply:
Possibly? I don't have a clear set of requirements for the export or import, so I've been working based on what the v1 serdes supports. (`User` not exported; no personal collections or anything in them by default; you can specify a user to get their personal collections added.)

I've been designing throughout to leave lots of room for later extensions, so it'll be easy to add this later if we ever need it.